schemaless/sources.py

```diff
@@ -698,6 +705,8 @@ class PermitAddendaSummary(Source):
         'permit_number',
         'earliest_addenda_arrival'
     ]
+    DATA_SF = ""  # TODO: Not public
+    DATA_SF_DOWNLOAD = ""  # TODO: Not public
```

Comment:
I think this is public?

Reply:
Oh, is it this dataset? https://data.sfgov.org/Housing-and-Buildings/Department-of-Building-Inspection-Permit-Addenda-w/87xy-gk8d

Reply:
Yup

Reply:
fixed